### ISSUE TITLE
Colored lights glow through blocks.

### ISSUE THREAD

**Issue by WbMystery** [Razzokk/Colored-Lights#18]
Hello there. Whenever I use this mod, the lights glow through blocks even if they're enclosed. I've made sure albedo is installed and tried it with multiple blocks and light colors.
![2020-04-12_10 51 44](https://user-images.githubusercontent.com/63551365/79073425-1febfd00-7cac-11ea-8e6b-99bdb8cccd0c.png)
![2020-04-12_10 51 53](https://user-images.githubusercontent.com/63551365/79073428-211d2a00-7cac-11ea-967a-5caa9b3602d9.png)

**Comment by Razzokk** [comment]
Yes that is a known issue from Albedo, can‘t change it for now. I‘m gonna remake the if I‘m able to modify OpenGL stuff in a way that colored lighting works without the need of third party libraries.
